community/kernel-api/src/test/java/org/neo4j/internal/kernel/api/RandomRelationshipTraversalCursorTestBase.java

```diff
@@ -40,7 +40,8 @@
     private static final int N_TRAVERSALS = 10_000;
     private static int N_NODES = 100;
     private static int N_RELATIONSHIPS = 1000;
-    private static Random random = new Random( 666 );
+    private static long seed = (new Random()).nextInt();
+    private static Random random = new Random( seed );
```

Comment:
Cool I didn't know about that! I prefer not to use it here though, since there is only one test and therefore not so much code duplication, while `RandomRule` would introduce a whole new dependency on the commons test-jar. If we need more test utils later, maybe this is an option.